stable_baselines3/ppo/ppo.py

```diff
@@ -1,3 +1,5 @@
+import warnings
+from math import floor
```

Comment:
you don't need that normally, for instance `3 // 2` is the euclidian division of 3 by 2 ;)

Reply:
Ah, right, I'd forgotten that functionality was still preserved there in Python3